static/redesign/apps/style/views/right/marker-style-view.js

```diff
@@ -44,15 +44,19 @@ define(["jquery",
 
             initialize: function (opts) {
```

Comment:
Great work on this refactor -- your instincts were right on. This file is readable and much easier to understand and test. Well done.